beerus_cli/src/ethereum/mod.rs

```diff
@@ -64,3 +64,20 @@ pub async fn query_block_number(beerus: BeerusLightClient) -> Result<CommandResp
     let block_number = beerus.ethereum_lightclient.get_block_number().await?;
     Ok(CommandResponse::EthereumQueryBlockNumber(block_number))
 }
+
+/// Query the code of a contract
+/// # Arguments
+/// * `beerus` - The Beerus light client.
+/// # Returns
+/// * `Result<CommandResponse>` - Vector of 256bits
+/// # Errors
+/// * If the block number query fails.
+pub async fn query_code(beerus: BeerusLightClient, address: String) -> Result<CommandResponse> {
+    let block = BlockTag::Latest;
```

Comment:
Can you make this configurable ? or open an issue to do it and add a `TODO`

Reply:
Ok!

Reply:
When u mean configurable would it be to receive it as an argument?

Reply:
yes